README.md

```diff
@@ -17,16 +17,16 @@ F´ (F Prime) is a component-driven framework that enables rapid development and
 - A growing collection of ready-to-use components
 - Testing tools for testing flight software at the unit and integration levels.
 
-Learn more about [F´ key features](https://nasa.github.io/fprime/features.html).
+Learn more about [F´ key features]([https://(https://fprime.jpl.nasa.gov/latest/)).
 
 
 ## System Requirements
 
-1. Linux, Windows with WSL, or macOS operating system
-2. git
+1. [Linux](https://www.debian.org/), Windows with WSL, or macOS operating system
+2. [git](https://git-scm.com/)
 3. [CMake 3.16+](https://cmake.org/download/). CLI tool must be available on the system path.
-4. Clang or GNU C and C++ compilers (e.g. gcc and g++)
-5. [Python 3.8+](https://www.python.org/downloads/), virtual environments, and PIP
+4. [Clang](https://clang.llvm.org/) or [GNU C and C++ compilers](https://gcc.gnu.org/) (e.g. gcc and g++)
+5. [Python 3.8+](https://www.python.org/downloads/), [virtual environments](https://www.virtualbox.org/), and [PIP](https://pypi.org/project/pip/)
```

Comment:
i highly doubt this "virtual environment" noun was meant to reference virtual machines like virtualbox. Likely means something like python virtualenv or conda. Also not sure how useful these links are.